Resources/Controllers/BottomSheet/README.md

```diff
@@ -0,0 +1,159 @@
+#  BottomSheet
+
+A UI component that effectively replaces and extends native UIActionSheet.  
```

Comment:
I think `Wizard` is a bit of an overkill as a replacement for UIActionSheet as it prepares a lot of stuff regarding animations between different steps and so on.